lingua_franca/res/text/en-us/normalize.json

```diff
@@ -99,7 +99,78 @@
     "you'd've": "you would have",
     "you'll": "you will",
     "you're": "you are",
-    "you've": "you have"
+    "you've": "you have",
+    "I'm'a": "I am about to",
+    "I'm'o": "I am going to",
+    "I'll've": "I will have",
+    "I'd've": "I would have",
+    "Whatcha": "What are you",
+    "amn't": "am not",
+    "'cause": "because",
+    "can't've": "cannot have",
+    "couldn't've": "could not have",
+    "daren't": "dare not",
+    "daresn't": "dare not",
+    "dasn't": "dare not",
+    "everyone's": "everyone is",
+    "gimme": "give me",
+    "gon't": "go not",
+    "hadn't've": "had not have",
+    "he've": "he have",
+    "he'll've": "he will have",
+    "he'd've": "he would have",
+    "here's": "here is",
+    "how're": "how are",
+    "how'd'y": "how do you",
+    "'tis": "it is",
+    "'twas": "it was",
+    "it'll've": "it will have",
+    "it'd've": "it would have",
+    "kinda": "kind of",
+    "let's": "let us",
+    "ma'am": "madam",
+    "may've": "may have",
+    "mayn't": "may not",
+    "mightn't've": "might not have",
+    "mustn't've": "must not have",
+    "needn't've": "need not have",
+    "ol'": "old",
+    "oughtn't've": "ought not have",
+    "sha'n't": "shall not",
+    "shalln't": "shall not",
+    "shan't've": "shall not have",
+    "she'd've": "she would have",
+    "shouldn't've": "should not have",
+    "so've": "so have",
+    "so's": "so is",
+    "something's": "something is",
+    "that're": "that are",
+    "that'd've": "that would have",
+    "there'll": "there will",
+    "there'd've": "there would have",
+    "these're": "these are",
+    "they'll've": "they will have",
+    "they'd've": "they would have",
+    "this's": "this is",
+    "this'll": "this will",
+    "this'd": "this would",
+    "those're": "those are",
+    "to've": "to have",
+    "wanna": "want to",
+    "we'll've": "we will have",
+    "we'd've": "we would have",
+    "what'll've": "what will have",
+    "when've": "when have",
+    "where're": "where are",
+    "which's": "which is",
+    "who'll've": "who will have",
+    "why've": "why have",
+    "will've": "will have",
+    "y'all're": "you all are",
+    "y'all've": "you all have",
+    "y'all'd": "you all would",
+    "y'all'd've": "you all would have",
+    "you'll've": "you shall have"
```

Comment:
"you will have" better matches others in this list (though I'm not sure one is "more" correct that the other in any case)